src/test/java/DukeTest.java

```diff
@@ -0,0 +1,30 @@
+import duke.*;
+import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
+
+    Duke duke = new Duke("C:\\Users\\Rickie\\Documents\\University\\Year 2\\Semester 1\\CS2103T\\ip\\Data\\taskList.txt");
+
+    @Test
+    public void parserTest1() {
```

Comment:
Perhaps need a descriptive header comments for the parserTest1 method? 🤔
And I noticed the same issue in several other places below.